pkg/node/node.go

```diff
@@ -254,6 +254,41 @@ func NewBee(ctx context.Context, addr string, publicKey *ecdsa.PublicKey, signer
 
 	addressbook := addressbook.New(stateStore)
 
+	pubKey, _ := signer.PublicKey()
+	if err != nil {
```

Comment:
I see that it was like this from before but maybe we can fix this now by either adding the `err` on line 257 or getting rid of the check altogether